pkgs/development/interpreters/clojurescript/lumo/deps.nix

```diff
@@ -0,0 +1,392 @@
+# generated by clj2nix
+let repos = [
+        "https://repo.clojars.org/"
+        "https://repo1.maven.org/"
+        "http://central.maven.org/maven2/"
+        "http://oss.sonatype.org/content/repositories/releases/"
+        "http://oss.sonatype.org/content/repositories/public/"
+        "http://repo.typesafe.com/typesafe/releases/"
+      ];
+  pkgs = import <nixpkgs> {};
```

Comment:
Ohh yeah, this is the problem here. Does clj2nix have a way to not use `import <nixpkgs> {}` and instead take an argument? Otherwise this needs to be fixed manually.

Reply:
oh shit, this is my first nixos plugin, so it's no surprise I did something wrong. Yes, I'll find a solution, inherit pkgs or worst case, pass fetchMavenArtifact as argument. Can this wait 24h?

Reply:
Or maybe I do this after eating (2H)

Reply:
You could just add `pkgs: ` at the top of this file, remove line 10 instead.

Reply:
No worries, stuff breaks all the time, I'm just glad @cocreature noticed it this early :)